light-prover/scripts/download_keys.sh

```diff
@@ -35,45 +47,59 @@ get_bucket_url() {
         echo "https://${NON_INCLUSION_40_BUCKET}.ipfs.w3s.link/${FILE}"
     elif [[ $FILE == combined_26_40_* ]]; then
         echo "https://${COMBINED_26_40_BUCKET}.ipfs.w3s.link/${FILE}"
+    elif [[ FILE == combined_32_40_* ]]; then 
+        echo "https://${COMBINED_32_40_BUCKET}.ipfs.w3s.link/${FILE}"
     elif [[ $FILE == combined_26_* ]]; then
         echo "https://${COMBINED_26_26_BUCKET}.ipfs.w3s.link/${FILE}"
-    elif [[ $FILE == append-with-proofs_* ]]; then
-        echo "https://${APPEND_WITH_PROOFS_BUCKET}.ipfs.w3s.link/${FILE}"
-    elif [[ $FILE == append-with-subtrees_* ]]; then
-        echo "https://${APPEND_WITH_SUBTREES_BUCKET}.ipfs.w3s.link/${FILE}"
-    elif [[ $FILE == address-append_* ]]; then
-        echo "https://${APPEND_ADDRESS_BUCKET}.ipfs.w3s.link/${FILE}"
-    elif [[ $FILE == update_* ]]; then
-        echo "https://${UPDATE_BUCKET}.ipfs.w3s.link/${FILE}"
+    elif [[ $FILE == append-with-proofs_26_* ]]; then
+        echo "https://${APPEND_WITH_PROOFS_26_BUCKET}.ipfs.w3s.link/${FILE}"
+    elif [[ $FILE == append-with-proofs_32_* ]]; then
+        echo "https://${APPEND_WITH_PROOFS_32_BUCKET}.ipfs.w3s.link/${FILE}"
+    elif [[ $FILE == append-with-subtrees_26_* ]]; then
+        echo "https://${APPEND_WITH_SUBTREES_26_BUCKET}.ipfs.w3s.link/${FILE}"
+    elif [[ $FILE == address-append_40_* ]]; then
+        echo "https://${APPEND_ADDRESS_40_BUCKET}.ipfs.w3s.link/${FILE}"
+    elif [[ $FILE == update_26_* ]]; then
+        echo "https://${UPDATE_26_BUCKET}.ipfs.w3s.link/${FILE}"
+    elif [[ $FILE == update_32_* ]]; then
+        echo "https://${UPDATE_32_BUCKET}.ipfs.w3s.link/${FILE}"
     fi
 }
 
 case "$1" in
     "light")
         SUFFIXES=(
             "inclusion_26:1 2 3 4 8"
+            "inclusion_32:1 2 3 4 8"
             "mainnet_inclusion_26:1 2 3 4 8"
             "non-inclusion_26:1 2 3 4 8"
             "non-inclusion_40:1 2 3 4 8"
             "combined_26:1_1 1_2 2_1 2_2 3_1 3_2 4_1 4_2"
             "combined_26_40:1_1 1_2 1_3 1_4 2_1 2_2 2_3 2_4 3_1 3_2 3_3 3_4 4_1 4_2 4_3 4_4"
+            "combined_32_40:1_1 1_2 1_3 1_4 2_1 2_2 2_3 2_4 3_1 3_2 3_3 3_4 4_1 4_2 4_3 4_4"
             "append-with-proofs_26:1 10"
+            "append-with-proofs_32:1 10"
             "append-with-subtrees_26:1 10"
             "update_26:1 10"
+            "update_32:1 10"
             "address-append_40:1 10"
```

Comment:
```suggestion
        SUFFIXES=(
            "inclusion_32:1 2 3 4 8"
            "mainnet_inclusion_26:1 2 3 4 8"
            "non-inclusion_26:1 2 3 4 8"
            "non-inclusion_40:1 2 3 4 8"
            "combined_26:1_1 1_2 2_1 2_2 3_1 3_2 4_1 4_2"
            "combined_32_40:1_1 1_2 1_3 1_4 2_1 2_2 2_3 2_4 3_1 3_2 3_3 3_4 4_1 4_2 4_3 4_4"
            "append-with-proofs_32:1 10"
            "update_32:1 10"
            "address-append_40:1 10"
```

can we remove the newly generated keys for height 26 which we won't use in tests?